u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -732,12 +740,19 @@ u2f.sign = function(appId, challenge, registeredKeys, callback, opt_timeoutSecon
  * @param {number=} opt_timeoutSeconds
  */
 u2f.sendSignRequest = function(appId, challenge, registeredKeys, callback, opt_timeoutSeconds) {
+  var reqId = ++u2f.reqCounter_;
+  u2f.callbackMap_[reqId] = callback;
+  var timeoutSeconds = (typeof opt_timeoutSeconds !== 'undefined' ?
+          opt_timeoutSeconds : u2f.EXTENSION_TIMEOUT_SEC);
+  var req;
+
+  if (u2f.isIOSChrome_()) {
```

Comment:
If you create a WrappedIosPort object instead, one that implements a postMessage function, this branch can disappear and become part of the normal branch below.